app/reconciler/index.ts

```diff
@@ -0,0 +1,27 @@
+import { executeChangeSet } from '~/lib/dns.server';
+import logger from '~/lib/logger.server';
+import readDbIntoCompareStructure from './readDbIntoCompareStructure.server';
+import readRuote53IntoCompareStructure from './readRuote53IntoCompareStructure.server';
+import createChangeSetFromCompareStructures from './createChangeSetFromCompareStructures';
+
+const reconciler = async () => {
+  const dbStructure = await readDbIntoCompareStructure();
+  const route53Structure = await readRuote53IntoCompareStructure();
```

Comment:
You might as well run these at the same time:

```ts
const [dbStructure, route53Structure] = Promise.all([readDbIntoCompareStructure(), readRuote53IntoCompareStructure()]);
```

Reply:
Good point